server/src/main/java/org/elasticsearch/http/CorsHandler.java

```diff
@@ -68,6 +68,11 @@
 public class CorsHandler {
 
     public static final String ANY_ORIGIN = "*";
+    public static final String ORIGIN = "origin";
+    public static final String DATE = "date";
+    public static final String VARY = "vary";
+    public static final String ACCESS_CONTROL_REQUEST_METHOD = "access-control-request-method";
```

Comment:
Should we use these constants in Netty4CorsHandler and NioCorsHandler implementations as well?